### ISSUE TITLE
Possible new build source with updates

### ISSUE THREAD

**Issue by Desani** [Jackysi/advancedtomato#406]
Hey Jacky,

I have recently been made aware of a Shibby Tomato fork called Fresh Tomato by kille72. Basically picking up where shibby left off, and adding a bunch of much needed security updates.

https://www.linksysinfo.org/index.php?threads/fork-freshtomato-arm.74117/

Would you consider putting the Advanced Tomato front end on this updated build? I refuse to use anything if it does not support Advanced Tomato.

Thanks,

**Comment by sstraus** [comment]
+1

**Comment by Natteverf** [comment]
I second that!

**Comment by april** [comment]
This gets my vote as well, as shibby is looking solidly like abandonware at this point.

**Comment by ghost** [comment]
Here's the same fork but for MIPS devices http://www.linksysinfo.org/index.php?threads/fork-freshtomato-mips.74145/

**Comment by dotjaz** [comment]
Good idea, since Shibby won't be updating his repo any time soon, new source is warranted.

**Comment by Jackysi** [comment]
True. But adapting to another firmware with so many changes is way too much for me at the time. Its A LOT of work. Also merging is almost impossible (attempted) without real manual work. So this is big process. 

When I have time. If ever that much. Otherwise help from community would be good (e.g. not having to work on this my self)

**Comment by ecc29** [comment]
At this moment FreshTomato is basically the same GUI as the last version of Shibby Tomato. They just forked from Shibby Tomato and put some security updates. I guess AdvancedTomato doesn't need to start from scratch. I heard Shibby is very busy because he has a new baby. So we cannot expect any updates for Shibby Tomato in the near future. I really hope AdvancedTomato will not go to obsolete because of this.

**Comment by botho** [comment]
+1
+1
+1

**Comment by radarlog** [comment]
agree

**Comment by FredericMa** [comment]
+1, would be very nice!

**Comment by nemchik** [comment]
I've added a :thumbup: to the OP up top like many others have already, and hit the subscribe button to be notified of updates here. Hopefully others can do the same to express interest and keep the "me too" and "+1" comments to a minimum.

@Jackysi has labeled this issue which seems to mean it's not off the table. I'm sure he will update us if/when this gains traction.

**Comment by HydrelioxGitHub** [comment]
Same thumb up here ...

**Comment by xbreaker** [comment]
+1

**Comment by hobbit19** [comment]
pls update firmare

**Comment by boktai1000** [comment]
I've noticed there are two different repositories / forks for FreshTomato, @kille72 and pedro. Which one would this be built off of, @kille72 I'm assuming?

**Comment by adriel** [comment]
@boktai1000 Neither, I believe it's a fork from Tomato by Shibby. Check out the [FAQ](https://advancedtomato.com/faq#versioning-explained).

**Comment by boktai1000** [comment]
@adriel oh I think you may have misunderstood my question, this issue is regarding building Advanced Tomato off a new base using FreshTomato instead of Tomato by Shibby. For FreshTomato, it seems there's two different versions/forks and I was just wondering which one of those two (kille72 or pedro) would be recommended to serve as the basis?

Maybe it doesn't matter - I'm not sure! Just something I was wondering about, as I imagine there are some slight differences between them. I think I'm getting ahead of myself a little bit though, if so my apologies! :)

**Comment by adriel** [comment]
@boktai1000 ooh oops, yeah that makes more sense, my bad.

I always thought pedro and kille72 were working on the same one?
That's what it looked like [here](https://www.linksysinfo.org/index.php?threads/fork-freshtomato-arm.74117/) "Forked off from Tomato-ARM by Shibby, builds compiled by FreshTomato team
FreshTomato team: @kille72 @pedro311"

I could be wrong though.

PS, I've been running this fork of FreshTomato for a few weeks now and it seems pretty rock solid, but I do really miss the clean UI of AdvancedTomato :(

**Comment by boktai1000** [comment]
@adriel to elaborate on what I meant regarding which fork / two different code-bases if you go to the forum post https://www.linksysinfo.org/index.php?threads/fork-freshtomato-arm.74117/ as you have seen the team consists of kille72 and Pedro311 but they each have their own BitBucket with different patches/etc it seems.

kill72 Source Code and Downloads: https://bitbucket.org/kille72/freshtomato-arm
Pedro311 Source Code and Downloads: https://bitbucket.org/pedro311/freshtomato-arm

I didn't really know much about FreshTomato when I made my initial comment, so I was originally wondering which of the sources it would make sense for Jackysi to base off of moving forward since each seem to cherry pick different commits, I didn't really understand the difference between each of them. But hopefully that makes more sense what I originally meant.

Update: I also did want to mention that I came across this - someone has merged AdvancedTomato and FreshTomato in an official firmware already located here: https://bitbucket.org/AndreDVJ/advancedtomato-arm/downloads/ - I originally saw this mentioned in the thread for FreshTomato here https://www.linksysinfo.org/index.php?threads/fork-freshtomato-arm.74117/page-13#post-300520

**Comment by ecc29** [comment]
@boktai1000, this thread has been here for more than 8 month. I doubt if @Jackysi will read it (if he has time). Maybe you can open a new issue to talk about your discovery, especially for the new fork made by AndreDVJ. In case this project is abandoned, at least people will know where to find alternatives.

**Comment by boktai1000** [comment]
@ecc29 done I have compiled what information I know of these builds as well as FreshTomato here: https://github.com/Jackysi/advancedtomato/issues/430 

Some users may not be aware of this if they only visit the Advanced Tomato website and maybe even this GitHub - I did not become aware of it until I did some digging in the comments and throughout the greater Tomato community.

**Comment by Jackysi** [comment]
Actually I read every comment. I just refuse to respond to them. Very quick google search or even reading previous comments explains situation. I don't have life time of time to waste on responding to hundreds of emails/comments/messages on forums. I understand people want answers, but they're available by search. If they don't care enough to search then why should I to answer?

https://github.com/Jackysi/advancedtomato/issues/430
